src/lib.rs

```diff
@@ -27,6 +54,10 @@ pub struct Criu {
     file_locks: Option<bool>,
     manage_cgroups: Option<bool>,
     work_dir_fd: i32,
+    freeze_cgroup: Option<String>,
+    cgroups_mode: Option<CgMode>,
+    cgroup_props: Option<String>,
+
```

Comment:
Please remove this empty line.

Reply:
> Please remove this empty line.
@adrianreber 
Thanks. I have performed a rebase, removed the blank lines, and executed a cargo clippy check.